website/docs/docs/build/projects.md

```diff
@@ -77,6 +77,12 @@ To use the Project subdirectory option in dbt Cloud, follow these steps:
 
 After configuring the Project subdirectory option, dbt Cloud will use it as the root directory for your dbt project. This means that dbt commands, such as `dbt run` or `dbt test`, will operate on files within the specified subdirectory. If there is no `dbt_project.yml` file in the Project subdirectory, you will be prompted to initialize the dbt project.
 
+:::info Multiple projects in dbt Cloud
```

Comment:
non blocker and sugg a diff title since the purpose of the callout is to inform users about the project availability based on plans
```suggestion
:::info Project support in dbt Cloud plans
```